ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -54,6 +56,53 @@ def vector_encoder(field_index):
     return vector_encoder
 
 
+local_int2byte = struct.Struct(">B").pack
```

Comment:
This was local in the original code, but it's not local here. Also, it's public for no reason.